crates/biome_configuration/src/editorconfig.rs

```diff
@@ -210,6 +219,32 @@ impl Diagnostic for EditorConfigValidationError {
         }
         Ok(())
     }
+
+    fn source(&self) -> Option<&dyn Diagnostic> {
+        match self {
+            EditorConfigValidationError::ParseFailed { source } => Some(source),
+            _ => None,
+        }
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, Diagnostic)]
+#[diagnostic(
+    category = "configuration",
+    severity = Error,
```

Comment:
I think it should be an error so that the user is forced to address it. Some people skip over warnings, and novice devs especially don't address warnings if they end up getting their desired result anyway.

Biome doesn't work if your `biome.json` is invalid. We are effectively treating `.editorconfig` as another place to put configuration, so we should match that behavior.

Reply:
Yeah it makes sense, especially if users rely on editor config. How does Prettier behave?

Reply:
https://github.com/prettier/prettier/blob/c121f98332cd8ad4f1482a1acc5413c61426b9dc/src/config/editorconfig/index.js#L21

At a glance, it looks like it just throws. It defers the parsing to the `editorconfig` package.